src/Symfony/Component/Cache/Tests/Simple/FilesystemCacheTest.php

```diff
@@ -22,4 +22,52 @@ public function createSimpleCache($defaultLifetime = 0)
     {
         return new FilesystemCache('', $defaultLifetime);
     }
+
+    public function testPrune()
```

Comment:
This is a valid point; do you have any idea how to better test this functionality?